src/filesystem.cpp

```diff
@@ -201,7 +201,7 @@ void DirNotify::notify(notifyqueue q, LocalNode* l, const char* localpath, size_
         delete fa;
     }
 
-    if (q == DirNotify::DIREVENTS)
+    if (q <= DirNotify::DIREVENTS)
```

Comment:
For readability and to avoid future conflicts with new queues we may add in the future, I'd change this condition to:
```if (q == DirNotify::DIREVENTS || q == DirNotify::EXTRA)```

Reply:
I agree :+1: